src/components/navigation-bar/ui/auth/AuthDropdownContainer.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC, ReactNode } from "react";
+import { useStyletron } from "baseui";
+import AuthAvatar from "./AuthAvatar";
+import { LabelSmall } from "baseui/typography";
+import { PRIMITIVE_COLORS } from "../../../../shared";
+import { LogoutIcon } from "../../../icons";
+import { Button, BUTTON_SIZE } from "../../../button";
+
+export type AuthDropdownContainerProps = {
+  username: string;
+  children?: ReactNode;
+  onLogout?: () => void;
+};
+
+const containerStyles = {
+  minWidth: "280px",
+};
+
+const wrapperStyles = {
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center",
+  padding: "16px",
+};
+
+const authWrapperStyles = {
+  display: "flex",
+  alignItems: "center",
+};
+
+const labelStyles = {
+  marginLeft: "8px",
+};
+
+const AuthDropdownContainer: FC<AuthDropdownContainerProps> = ({ username, children, onLogout }) => {
```

Comment:
i need opt-out of default element here. Like if i don't provide username it should skip all part. Because in DBMS i want to have another type of displaying user data.